File: backend/src/routes/health.py
"""Health check endpoint."""
from fastapi import APIRouter, Depends

from src.config import Settings, get_settings

router = APIRouter()


@router.get("/health")
async def health_check(settings: Settings = Depends(get_settings)) -> dict:
    """Return health status of the service."""
    return {
        "status": "ok",
        "version": settings.VERSION,
    }

File: backend/src/routes/health.js
const express = require('express');
const router = express.Router();

const VERSION = '1.0.0';

/**
 * Health check endpoint
 * GET /health
 * Response: { status: "ok", timestamp: "ISO8601", version: "1.0.0", correlationId: "uuid" }
 */
router.get('/health', (req, res) => {
  res.json({
    status: 'ok',
    timestamp: new Date().toISOString(),
    version: VERSION,
    correlationId: req.correlationId
  });
});

module.exports = router;
